### PR TITLE
Fix/42679 - Reduce metrics cardinality

### PR DESCRIPTION
**Pull request by mjk90**
## Description

- Removed request_origin label from metrics to reduce metrics cardinality. The IP & hostname are being logged for each request so it's not worth it to keep it as a metrics label if it increases the size of metrics data too much.
- Added 'result' to excluded properties in cache key & feed ID generation

## Changes

- Removed request_origin label from metrics & tests
- Added 'result' to excluded properties in cache key & feed ID generation

## Steps to Test

1. `yarn test:unit`
2. `yarn test:integration`
3. Run an EA locally with metric enabled & check that request_origin label is no longer present in metrics

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

**Comment by justinkaseman** [comment]
Do we know 100% that this was the part causing the problem? Like do we have a before vs. after benchmark?

**Comment by justinkaseman** [comment]
It seems to me like request origin should be relatively stable.

**Review by boxhock — DISMISSED** [review]

**Comment by mjk90** [comment]
> Instead of outright removing it, have you considered just removing the outbound port?
> 
> If that's not a viable option or if the request origin is not needed in metrics, then this PR LGTM

I did consider that but I though the overhead of finding and replacing a string on each request would probably not be worth it

File: .changeset/twelve-queens-sell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ea-bootstrap': minor
+---
+
+Removed request_origin label from metrics
```
Comment (boxhock) [inline .changeset/twelve-queens-sell.md:5]: Should add a comment about the `result` key being removed as welll

**Review by boxhock — CHANGES_REQUESTED** [review]
One small comment for the changeset, otherwise LGTM

**Comment by justinkaseman** [comment]
Not for this PR, but something I am thinking about is how we can better catch issues like this in the future. Maybe setting up an alert on the cardinality of our metrics with something like https://github.com/thought-machine/prometheus-cardinality-exporter

**Pull request merged**